packages/components/pie-button/README.md

```diff
@@ -40,12 +40,12 @@ cd packages/components/pie-button
 yarn build
 ```
 
-compile using Vite (auto-compiles `dist` on save)
+Compile and watch for changes (auto-compiles `dist` on save)
 ```
 yarn watch
 ```
 
-Local dev server using Vite (with hot module reloading)
+Run local dev server (with hot module reloading)
```

Comment:
I think we can remove this. I don't actually think this does anything unless it's required for Storybook? Simply running `yarn dev` just links to http://localhost:5173/ and says the page cannot be found.

Reply:
Yep happy for that to be removed